src/hb-subset-plan.hh

```diff
@@ -38,9 +38,14 @@ struct hb_subset_plan_t {
   // TODO(Q1) actual map, drop this crap
   // Look at me ma, I'm a poor mans map codepoint : new gid
   // codepoints is sorted and aligned with gids_to_retain.
+  // TODO Also you should init/fini those arrays
```

Comment:
I believe I did that, humm.

Reply:
Oh so you did, my bad. I had a note to remember to add it.